src/component/marker/MarkLineView.js

```diff
@@ -67,9 +67,12 @@ var markLineTransform = function (seriesModel, coordSys, mlModel, item) {
         mlTo.coord[baseIndex] = Infinity;
 
         var precision = mlModel.get('precision');
-        if (precision >= 0 && typeof value === 'number') {
+        if (precision && precision >= 0 && typeof value === 'number') {
             value = +value.toFixed(Math.min(precision, 20));
         }
+        if (precision == null && mlType === 'average') {
+            value = +value.toFixed(2);
```

Comment:
I'm not sure. But it seems a little odd to me that the average shows 2 digits by default while the median shows something like `0.12345678`.